modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-field-type/src/main/resources/META-INF/resources/DatePicker/DatePicker.es.js

```diff
@@ -368,37 +375,52 @@ const Main = ({
 	predefinedValue,
 	readOnly,
 	spritemap,
+	type,
 	value,
 	...otherProps
-}) => (
-	<FieldBase
-		{...otherProps}
-		localizedValue={localizedValue}
-		name={name}
-		readOnly={readOnly}
-		spritemap={spritemap}
-	>
-		<DatePicker
-			defaultLanguageId={defaultLanguageId}
-			disabled={readOnly}
-			formatInEditingLocale={
-				localizedValue &&
-				localizedValue[locale] !== undefined &&
-				localizedValue[locale] !== null
-			}
-			locale={locale}
-			localizable={localizable}
+}) => {
+	let use12Hours;
+
+	const isDateTime = type === 'date_time';
+
+	if (isDateTime) {
+		const momentLocale = moment().locale(locale);
+		const time = momentLocale.localeData().longDateFormat('LT');
+		use12Hours = time.endWith('A');
+	}
```

Comment:
Hi @rebsilva, sorry for the typo on the documentation. Itś missing an 's' on endWith method. Please, change `time.endWith('A')`  by `time.endsWith('A');`